### PR TITLE
Improving hand documentation

### PR DESCRIPTION
**Pull request by beatrizleon**
## Proposed changes

Updated the menus, removed bimanual and divided some pages to make it more clear and easy to reference.

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [x] ~~Manually tested that added code works as intended (if any functional/runnable code is added).~~
- [x] ~~Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).~~
- [x] ~~Tested on real hardware (if the changed or added code is used with the real hardware).~~
- [x] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

**Review by joannaphillips45 — APPROVED** [review]

File: docs/_static/css/custom.css [hunk 1]
```diff
@@ -1,3 +1,25 @@
+h1 {
+  /*color: #D30074;*/
```
Comment (mykolasjuraitis) [inline docs/_static/css/custom.css]: Why commented? Maybe can be deleted?

File: docs/user_guide/1_1_pelicase.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
++-----------------------------------------------------------------------+--------------------------------------+
+| Toolbox (Contains hex drivers to robot maintenance)                   |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Cut allen key (inside the Toolbox)                                    |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Allen key (inside the Toolbox)                                        |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Spooltool (inside the Toolbox)                                        |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Luggage locks for pelicase                                            |     2                                |
```
Comment (mykolasjuraitis) [inline docs/user_guide/1_1_pelicase.rst]: In the tile `pelicase` is spelled using 2 words. Which one is the correct one?

File: docs/user_guide/1_4_simulated_hand_mujoco.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
+Mujoco
+=======
+
+`Mujoco <http://www.mujoco.org/>`_ is a robot simulator that has now been adopted by a wide community of researchers and developers, specially for
+machine learning applications. We have developed the tools and the model of our dexterous hand to use Mujoco as an alternative to Gazebo. 
+Mujoco is not free so follow the next instructions if you have already a `Mujoco License <https://www.roboti.us/license.html>`_.
+
+
+Obtaining the mujoco simulation
```
Comment (mykolasjuraitis) [inline docs/user_guide/1_4_simulated_hand_mujoco.rst]: `mujoco` should start with a capital letter as in other places.

File: docs/user_guide/2_connecting_the_cables.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
+Connecting Cables
+------------------
+
+* Connect the hands to the NUC-CONTROL. It is very important that the exact USB-ethernet adapters are used.
+
+  * The right hand should be connected to a USB->ethernet adapter labelled: ``HAND RIGHT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
```
Comment (mykolasjuraitis) [inline docs/user_guide/2_connecting_the_cables.rst]: You start word `ethernet` with capital letter bellow. Maybe here too for consistency?

File: docs/user_guide/2_connecting_the_cables.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+Connection procedure
+^^^^^^^^^^^^^^^^^^^^^
+1. Connect the ethernet between the NUC and the laptop using the instructions above
+2. Power on the laptop
+3. Connect an ethernet cable providing external internet connection to the back of the laptop
+4. Power on the NUC
+5. Make sure the laptop has only 1 USB-Ethernet adapter connected to it.
+6. In case of using another laptop than one provided, please follow the instructions below to install the software.
+7. Power on the hand(s)
+8. Connect the right hand to the USB-ethernet adapter labelled “HAND RIGHT” which should be plugged in to the NUC, as explained above
```
Comment (mykolasjuraitis) [inline docs/user_guide/2_connecting_the_cables.rst]: Above you use `USB->ethernet` but here `USB-ethernet`. Should it be the same everywhere?

File: docs/user_guide/1_laptop_box.rst [hunk 6]
```diff
@@ -0,0 +1,17 @@
+Laptop box contents 
+-------------------------------------------------
+This is an extra box additional to the hand(s) peli case(s). It will only be 1 per bimanual system..
```
Comment (carebare47) [inline docs/user_guide/1_laptop_box.rst]:
```suggestion
This is an extra box, additional to the hand(s) peli case(s). There will be 1 per bimanual system.
```

File: docs/user_guide/2_connecting_the_cables.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+* Connect the hands to the NUC-CONTROL. It is very important that the exact USB-ethernet adapters are used.
+
+  * The right hand should be connected to a USB->ethernet adapter labelled: ``HAND RIGHT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
+  * The left hand should be connected to a USB->ethernet adapter labelled: ``HAND LEFT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
+
+* Connect one USB->Ethernet adapter labelled ``NUC-CONTROL`` to another USB port on the NUC and the other USB->Ethernet labelled ``SERVER`` to any of the ports in your ``SERVER`` Laptop (provided by Shadow or a custom one).
+* Connect the two adaptors together with an Ethernet cable.
+
+You have been supplied with medium length Ethernet leads, but if you require a different length, you can simply use a standard commercial Ethernet Cat 5 cable, available from most computer parts suppliers. If you require internet connection in the laptop, connect an ethernet cable providing external internet connection to the back of the laptop, to an ethernet port labelled ``INTERNET``.
```
Comment (carebare47) [inline docs/user_guide/2_connecting_the_cables.rst]:
```suggestion
You have been supplied with medium length Ethernet leads, but if you require a different length, you can simply use a standard commercial Ethernet Cat 5 (or better) cable, available from most computer parts suppliers. If you require internet connection in the laptop, connect an ethernet cable providing external internet connection to the back of the laptop, to an ethernet port labelled ``INTERNET``.
```

File: docs/user_guide/1_1_pelicase.rst (hunk 2 above)
Comment (maxzieba) [reply docs/user_guide/1_1_pelicase.rst]: and "Contains" with lower case maybe?

File: docs/user_guide/3_installing_the_software.rst [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+| Graphics Card | Nvidia GPU (optional)                                                                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| LAN           | A spare LAN port to connect the Hand (even with a USB to LAN adaptor)                                      |
++---------------+------------------------------------------------------------------------------------------------------------+
+| OS            | Ubuntu 20.04 Noetic (Active development), 18.04 Melodic, 16.04 Kinetic or 14.04 Indigo for older releases. |
++---------------+------------------------------------------------------------------------------------------------------------+
+
+The most important one is to have a fast HDD or an SSD.
+
+We have created a one-liner that is able to install Docker, download the docker image and create a new container for you.
```
Comment (mykolasjuraitis) [inline docs/user_guide/3_installing_the_software.rst]: Inconsistent capitalisation of `Docker`. 2 versions used even in this single sentence. Or is it deliberate that Docker as a tool is proper noun but docker type image is not?

File: docs/user_guide/3_installing_the_software.rst [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, follow these instructions:
+
+  1. Connect the ethernet between the NUC-CONTROL and the new PC using the instructions above
+  2. Power on the new PC
+  3. Connect an ethernet cable providing external internet connection to the back of the new PC
+  4. Power on the NUC-CONTROL
+  5. Install the hand software on the new PC by running the following on a terminal (Ctrl+Alt+T):
+
+ROS Melodic (Recommended) for a Right Hand:
```
Comment (mykolasjuraitis) [inline docs/user_guide/3_installing_the_software.rst]: Shouldn't `Noetic` be recommended now?

File: docs/_static/css/custom.css (hunk 1 above)
Comment (beatrizleon) [reply docs/_static/css/custom.css]: It is for the new color scheme, but when they allow us to use it, we can change again.

File: docs/user_guide/1_1_pelicase.rst (hunk 2 above)
Comment (beatrizleon) [reply docs/user_guide/1_1_pelicase.rst]: both done

File: docs/user_guide/1_4_simulated_hand_mujoco.rst (hunk 3 above)
Comment (beatrizleon) [reply docs/user_guide/1_4_simulated_hand_mujoco.rst]: done

**Review by carebare47 — APPROVED** [review]

File: docs/user_guide/2_connecting_the_cables.rst (hunk 4 above)
Comment (beatrizleon) [reply docs/user_guide/2_connecting_the_cables.rst]: I have replaced in all the file

File: docs/user_guide/2_connecting_the_cables.rst (hunk 5 above)
Comment (beatrizleon) [reply docs/user_guide/2_connecting_the_cables.rst]: fixed

File: docs/user_guide/3_installing_the_software.rst (hunk 8 above)
Comment (beatrizleon) [reply docs/user_guide/3_installing_the_software.rst]: fixed, leaved all with capital

File: docs/user_guide/3_installing_the_software.rst (hunk 9 above)
Comment (beatrizleon) [reply docs/user_guide/3_installing_the_software.rst]: Added noetic now and made it the recommended option.

**Comment by beatrizleon** [comment]
Thanks @mykolasjuraitis for your detailed review and also @carebare47 and @maxzieba. I fixed all so I will merge this and create new ones for other products.

**Pull request merged**